test/query_test.exs

```diff
@@ -65,6 +65,26 @@ defmodule QueryTest do
     assert [[[[0]]]] = query("SELECT ARRAY[ARRAY[0]]", [])
   end
 
+  test "decode array domain", context do
+    floats = [1.0, 2.0, 3.0]
+    assert :ok = query("DELETE FROM floats", [])
```

Comment:
Is it possible to make these tests pure, i.e. without a table: `SELECT ARRAY[1,2,3]::floats_domain` and `SELECT $1::floats_domain::text`?